modules/op-using-private-repositories-with-pipelines-as-code.adoc

```diff
@@ -52,5 +52,4 @@ tasks:
 ----
 <1> The `git-clone` task picks up the `basic-auth` workspace and uses it to clone the private repository.
 
-You can modify this configuration by setting the `secret-auto-create` flag to either a `false` or `true` value, as required in the {pac} config map.
-
+You can modify this configuration by setting the `secret-auto-create` flag to either a `false` or `true` value, as required, in the `TektonConfig` custom resource, under `pipelinesAsCode` and `settings`.
```

Comment:
here, flag is used - again I'd choose either flag, parameter, spec, field, etc etc and stick with one consistently throughout the docs

same comment re `pipelinesAsCode.settings` spec